content/en/blog/2023/otel-end-user-discussions-february-2023.md

```diff
@@ -0,0 +1,187 @@
+---
```

Comment:
Rename the file to `end-user-discussions-february.md`

Reply:
Or even `end-user-discussions-02.md`

Reply:
Done @chalin

Reply:
👍